test/functionalTest/cases/1715_batch_update/batch_update_error_entity_not_found.test

```diff
@@ -87,41 +87,41 @@ echo
 01. Batch Update with actionType UPDATE on non-existing entity, see error
 =========================================================================
 HTTP/1.1 404 Not Found
-Content-Length: 38
+Content-Length: 61
 Content-Type: application/json
 Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Date: REGEX(.*)
 
 {
-    "description": "E",
+    "description": "No context element found",
```

Comment:
Description changed to align with other messages. No need to mention this in CNR, from my point of view.

NTC (just a remark).